src/app/zap-templates/zcl/data-model/chip/unit-localization-cluster.xml

```diff
@@ -30,10 +30,15 @@ limitations under the License.
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
     <description>Nodes should be expected to be deployed to any and all regions of the world. These global regions
-      may have differing preferences for the units in which values are conveying in communication to a
+      may have differing preferences for the units in which values are conveyed in communication to a
       user. As such, Nodes that visually or audibly convey measurable values to the user need a
       mechanism by which they can be configured to use a user’s preferred unit.</description>
-    <!-- Base data types -->      
-    <attribute side="server" code="0x00" define="TEMPERATURE_UNIT" type="ENUM8" writable="true" optional="true">TemperatureUnit</attribute>
+    <globalAttribute side="either" code="0xFFFD" value="1"/>
+    <!-- Base data types -->
+    <attribute side="server" code="0x00" define="TEMPERATURE_UNIT" type="TempUnit" writable="true" optional="true">TemperatureUnit</attribute>
```

Comment:
TempUnit is not defined anywhere, right?

Reply:
It's defined above, in this file.

Reply:
Oh, indeed. I missed that somehow.  My apologies.